code/src/styled-components.js

```diff
@@ -0,0 +1,144 @@
+import styled from 'styled-components/macro';
+
+export const Container = styled.div`

```

Comment:
By using one big style components file it makes the components look clean and easy to read. Also in terms of reusability.